src/styles/components/_autoSuggest.scss

```diff
@@ -38,12 +38,12 @@
     }
 
     &--shifted {
-      padding-top: 10px;
+      padding-top: 13px;
```

Comment:
Why is this required? Wasn't it working fine?

Reply:
It was congested earlier so updated that.